sys-apps/lshw/files/lshw-02.18b-musl.patch

```diff
@@ -0,0 +1,58 @@
+diff -aurP a/src/core/dasd.cc b/src/core/dasd.cc
```

Comment:
Please tell us more about the patch origin.

Reply:
https://github.com/lyonel/lshw/commit/cd690bff1516b40fecd5ec4a7f6619e5bffc3cf0
https://patchwork.ozlabs.org/patch/655854/
^ Based on some changes made in these patches, adjusted.

Reply:
@hanetzer Patch origin and reason should be explained in the patch header

Reply:
Sorry, I have to reject this patch. Please see @vapier comment in https://github.com/lyonel/lshw/pull/19

Reply:
just copy the `c_str()` using `strcpy` into a temporary, muck with that, and dont forget to deallocate it afterwards